src/test/java/io/confluent/connect/elasticsearch/ElasticsearchClientTest.java

```diff
@@ -565,6 +566,102 @@ public void testNoVersionConflict() throws Exception {
     client2.close();
   }
 
+  @Test
+  public void testIncompatibleESVersionDataStreamNotSet() throws Exception {
```

Comment:
these tests never call the validator, they should be done in `ValidatorTest` or in ITs

Reply:
I have a set of tests like them in `ValidatorTest` that checks for the error message. The tests in this class were meant to check that records could still be written. I'll move them to the ITs.